src/components/baseButton/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./BaseButton";
```

Comment:
Prefer using named exported values instead of default
```suggestion
export { BaseButton } from "./BaseButton";
```